color-identifiers-mode.el

```diff
@@ -101,9 +102,10 @@ error colors etc."
   "Alist of major modes and the ways to distinguish identifiers in those modes.
 The value of each cons cell provides four constraints for finding identifiers.
 A word must match all four constraints to be colored as an identifier.  The
-value has the form (IDENTIFIER-CONTEXT-RE IDENTIFIER-RE IDENTIFIER-FACES
+value has the form (MAJOR-MODE IDENTIFIER-CONTEXT-RE IDENTIFIER-RE IDENTIFIER-FACES
```

Comment:
I had originally intended the term "value" to refer to the cdr of the cons cell, but perhaps that's not a standard usage. Still, keeping the word "value" seems confusing. How about replacing it with simply "the cons cell"?

```suggestion
cons cell has the form (MAJOR-MODE IDENTIFIER-CONTEXT-RE IDENTIFIER-RE IDENTIFIER-FACES
```

Reply:
Thanks, done! I agree it would be clearer. I thought of something like "element", but a "cons cell" sure suits too since we're talking about lisp.